api/v1beta1/cloudstackaffinitygroup_types.go

```diff
@@ -46,7 +48,7 @@ type CloudStackAffinityGroupStatus struct {
 
 //+kubebuilder:object:root=true
 //+kubebuilder:subresource:status
-
+//+k8s:conversion-gen=false
```

Comment:
don't we want conversion on affinity group?

Reply:
This is auto-generation of conversion methods. Yufei is hand writing the conversion, so we don't want conversion-gen to write these.